railway/utils/railway_client.py

```
@@ -0,0 +1,166 @@
+            result = subprocess.run(['railway', '--version'], 
+                                  capture_output=True, text=True, timeout=10)
+            return result.returncode == 0
+        except Exception:
+            return False
+    
+    def login_if_needed(self) -> bool:
+        """必要に応じてログイン"""
+        try:
+            # ログイン状態をチェック
+            result = subprocess.run(['railway', 'whoami'], 
+                                  capture_output=True, text=True, timeout=10)
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)